Dockerfile

```diff
@@ -53,4 +53,6 @@ WORKDIR /dscp-node
 
 ENTRYPOINT [ "/dscp-node/dscp-node" ]
 
-EXPOSE 30333 9933 9944
+HEALTHCHECK CMD curl -f http://localhost:9944/health || exit 1
+
+EXPOSE 30333 9615 9944
```

Comment:
Note this removes the old `ws` port that the node no longer listens on but documents the exposure of the prometheus telemetry port

Reply:
is anything else affected by this change or just prometheus?